pkg/backends/datadog/datadog_test.go

```diff
@@ -195,3 +196,79 @@ func metricsOneOfEach() *gostatsd.MetricMap {
 		},
 	}
 }
+
+func TestSendPercentileMetrics(t *testing.T) {
+	t.Parallel()
+	mux := http.NewServeMux()
+	mux.HandleFunc("/api/v1/series", func(w http.ResponseWriter, r *http.Request) {
+		data, err := ioutil.ReadAll(r.Body)
+		if !assert.NoError(t, err) {
+			return
+		}
+		enc := r.Header.Get("Content-Encoding")
+		if enc == "deflate" {
+			decompressor, err := zlib.NewReader(bytes.NewReader(data))
+			if !assert.NoError(t, err) {
+				return
+			}
+			data, err = ioutil.ReadAll(decompressor)
+			assert.NoError(t, err)
+		}
+		expected := `{"series":[` +
+			`{"host":"h2","interval":1.1,"metric":"t1.lower","points":[[100,0]],"tags":["tag2"],"type":"gauge"},` +
+			`{"host":"h2","interval":1.1,"metric":"t1.upper","points":[[100,0]],"tags":["tag2"],"type":"gauge"},` +
+			`{"host":"h2","interval":1.1,"metric":"t1.count","points":[[100,0]],"tags":["tag2"],"type":"gauge"},` +
+			`{"host":"h2","interval":1.1,"metric":"t1.count_ps","points":[[100,0]],"tags":["tag2"],"type":"rate"},` +
+			`{"host":"h2","interval":1.1,"metric":"t1.mean","points":[[100,0]],"tags":["tag2"],"type":"gauge"},` +
+			`{"host":"h2","interval":1.1,"metric":"t1.median","points":[[100,0]],"tags":["tag2"],"type":"gauge"},` +
+			`{"host":"h2","interval":1.1,"metric":"t1.std","points":[[100,0]],"tags":["tag2"],"type":"gauge"},` +
+			`{"host":"h2","interval":1.1,"metric":"t1.sum","points":[[100,0]],"tags":["tag2"],"type":"gauge"},` +
+			`{"host":"h2","interval":1.1,"metric":"t1.sum_squares","points":[[100,0]],"tags":["tag2"],"type":"gauge"},` +
+			`{"host":"h2","interval":1.1,"metric":"t1.buckets","points":[[100,5]],"tags":["tag2","between:0_20"],"type":"count"},` +
```

Comment:
Does anyone know why all the point arrays start with 100?

Reply:
It's the timestamp:
```
	client.now = func() time.Time {
		return time.Unix(100, 0)
	}
```